Added interactive example

estelle opened the pull request:
https://github.com/mdn/interactive-examples/pull/2175/ was merged, so added the interactive example to the page

estelle committed:
Added interactive example

wbamberg commented:
The "Try it" H2 that the interactive example adds means that prose content after the example and before "Syntax" is weirdly in a section called "Try it". It would be better to put this prose in a "Description" section after "Syntax", as we do for properties that need this: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types/CSS_property_page_template.

estelle committed:
add live example

estelle committed:
Merge pull request #324 from estelle/modal

add live example

estelle committed:
moved syntax up

estelle commented:
> The "Try it" H2 that the interactive example adds means that prose content after the example and before "Syntax" is weirdly in a section called "Try it". It would be better to put this prose in a "Description" section after "Syntax", as we do for properties that need this: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types/CSS_property_page_template.

Added  a description title and pushed the syntax section up.

I also added the :modal live example

wbamberg reviewed (approved):
👍 thanks @estelle !
